src/ripple/rpc/handlers/Fee1.cpp

```diff
@@ -30,7 +30,8 @@ namespace ripple
     {
         // Bail if fee escalation is not enabled.
         auto const view = context.app.openLedger().current();
-        if (!view || !view->rules().enabled(featureFeeEscalation))
+        BOOST_ASSERT(view);
```

Comment:
Why a `BOOST_ASSERT` if we have the `if` below? Also is `rpcNOT_ENABLED` the right error message now?

Reply:
I tend to overprotect with "belt and suspenders". Long story short, when I dug in to that function, it looked like `view` should never be null, so I asserted, but because it would lead to a crash triggered by user action if it ever was, I added the check. If we keep both, the error should definitely be changed to `rpcINTERNAL` now, and the check should be moved to `doRPC` itself. Do you think the `BOOST_ASSERT` is sufficient?